samples/snippets/csharp/tutorials/inheritance/shape.cs

```diff
@@ -64,7 +64,7 @@ public Circle(double radius)
       Radius = radius;
    } 
 
-   public override double Area => Math.Round(Math.Pow(Math.PI * Radius, 2), 2); 
+   public override double Area => Math.Round(Math.PI * Radius * Radius, 2); 
```

Comment:
Shouldn't we instead keep using the Pow method?
public override double Area => Math.Round(Math.PI * Math.Pow(Radius,2), 2);